src/main/java/io/github/enkarin/bookcrossing/books/controllers/MyBookController.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Objects;
 import java.util.Optional;
 
+
```

Comment:
This empty line looks redundant. I'd recommend to exclude corresponding checkstyle rule